src/messages.js

```diff
@@ -51,6 +51,9 @@ mixitup.Messages = function() {
     this.ERROR_FILTER_INVALID_ARGUMENTS =
         '[MixItUp] Please provide either a selector or collection `.filter()`, not both';
 
+    this.ERROR_SORT_NOT_PREEXISTING_ELEMENT =
+        '[MixItUp] An element to be sorted does not already exists in the container';
```

Comment:
Please change to `[MixItUp] An element to be sorted does not already exist in the container`